README.md

```diff
@@ -1,2 +1,71 @@
-# reviews-service
-Repo for the Airbnb reviews component
+# Air Tree n Tree Reviews Service
+
+---
+
+## API Spec
+
+`GET /api/reviews/:roomid/?review_group=[0 || selected pagination]`
+
+### Parameters
+
+| Params | Type |
+| --- | --- |
+| :roomId | `String` |
+| selected pagination | `Number` |
+
+### Response:
+
+| Field | Type |
+| ----- | ---- |
+| roomId | `Number` | 
+| reviews| [`Reviews[]`](#Reviews) |
+| averageRating| [`AverageRating{}`](#Average-Rating)|
```

Comment:
These links are not working. I think the links must match the header.

This might work [Reviews[]](#Response-Property:-Reviews)
I googled somewhere that the created urls are dash seperated.